tools/DDA/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ if(DEFINED IN_SOURCE_BUILD)
 else()
     add_executable( dvf dda.cpp )
 
-    target_link_libraries( dvf Svf Cudd ${llvm_libs} )
+    target_link_libraries( dvf Svf Cudd ${llvm_libs} pthread)
```

Comment:
Not sure why include pthread? DDA or Saber never used threads..

Reply:
It's not used by DDA and Saber, but by the libz3.

After testing the current master tree, this problem doesn't exist again.

Reply:
I currently think the reason is some components of SVF-2.4 use some definitions in `libz3.a(scoped_timer.o)`, which cause linker to link this `scoped_timer.o` object (as linker link project in object-granuality), here is part defined symbol in `scoped_timer.o`, 

![image](https://user-images.githubusercontent.com/35480440/174730431-b525852c-4d9d-4ca5-8ec4-46d27f39f8c3.png)

and then I search them in `saber` binary, it's included

![image](https://user-images.githubusercontent.com/35480440/174730685-a5ee9947-b877-441f-bb0f-bc56842b7e63.png)

However, the current master tree doesn't link `scoped_timer.o`.

![image](https://user-images.githubusercontent.com/35480440/174730954-af9a2d5c-140f-4666-b904-f83533f29cf7.png)

Reply:
Since there is no problem in newest master tree, I will try to use the newest one